modules/jaeger-install.adoc

```diff
@@ -0,0 +1,48 @@
+////
+This PROCEDURE module included in the following assemblies:
+- rhbjaeger-installation.adoc
+////
+
+[id="jaeger-operator-install_{context}"]
+= Installing the Jaeger Operator
+
+To install Jaeger you use the link:https://operatorhub.io/[OperatorHub] to install the Jaeger Operator.
+
+By default the Operator is installed in the `openshift-operators` project.
+
+.Prerequisites
+* Access to the {product-title} web console.
+* An account with the `cluster-admin` role.
+* If you require persistent storage, you must also install the Elasticsearch Operator before installing the Jaeger Operator.
+
+[WARNING]
+====
+Do not install Community versions of the Operators. Community Operators are not supported.
+====
+
+.Procedure
+
+. Log in to the {product-title} web console as a user with the `cluster-admin` role.
+
+. Navigate to *Operators* -> *OperatorHub*.
+
+. Type *Jaeger* into the filter box to locate the Jaeger Operator.
+
+. Click the *Jaeger Operator* provided by Red Hat to display information about the Operator.
```

Comment:
"Jaeger Operator" -> whatever we want to call the product, e.g. "Build of Jaeger" should possibly appear in the card instead of "Jaeger Operator", as it is obvious that it is an operator being in the operatorhub.